src/spec_concordius.cpp

```diff
@@ -0,0 +1,85 @@
+/************************************************************************/
+/*  Xania (M)ulti(U)ser(D)ungeon server source code                     */
+/*  (C) 2020 Xania Development Team                                     */
+/*  See the header to file: merc.h for original code copyrights         */
+/************************************************************************/
+
+#include "CHAR_DATA.hpp"
+#include "DescriptorFilter.hpp"
+#include "DescriptorList.hpp"
+#include "comm.hpp"
+#include "merc.h"
+#include "string_utils.hpp"
+#include <fmt/format.h>
+#include <string_view>
+#include <vector>
+
+using namespace fmt::literals;
+namespace {
+
+static inline constexpr auto SOCIAL_DICE = 30u;
+static inline constexpr auto NUM_SOCIAL_ACTIONS = 7u;
+
+inline static const std::vector<std::string> emotes{
```

Comment:
nice! Super minor: This can just be:

```
std::array emotes{"...", "..."};
```

(I learned the types are deduced in this case recently...) these are all constant strings, so `std::string` is overkill too, as written above this would be a std::array of const char*, but drop a `"$n tightens..."sv` `sv` suffix (with `using namespace std::literals`) to get an array of string_views, though either work. As does this in its current form!

Reply:
done